.circleci/workflows.yml

```diff
@@ -32,7 +32,7 @@ parameters:
     description: "Release Type. Accepted values [ Major, Minor, Patch ]"
     type: enum
     enum: ["Major", "Minor", "Patch"]
-    default: "Patch"
+    default: "Minor"
```

Comment:
patch will need to be modified here before the CI pipeline will stop trying to change the value for you